spaceship.zsh

```diff
@@ -88,6 +94,55 @@ spaceship_time() {
   echo -n "%{$reset_color%} "
 }
 
+# Battery
+spaceship_battery(){
+  [[ $SPACESHIP_BATTERY_SHOW == false ]] && return
+
+  local os=$(uname);
+  local battery_status="";
+  local battery_percent="0";
+
+  if [[ "$os" = "Linux" ]]; then
```

Comment:
Actually that's pretty big part of code. Maybe is there a way to use [battery plugin from OMZ](https://github.com/robbyrussell/oh-my-zsh/blob/master/plugins/battery/battery.plugin.zsh)?